web/client/utils/__tests__/LocaleUtils-test.js

```diff
@@ -80,4 +80,17 @@ describe('LocaleUtils', () => {
         expect(LocaleUtils.getLocalizedProp('it-IT', localizedObjectProp)).toBe('titolo');
         expect(LocaleUtils.getLocalizedProp('fr-FR', localizedObjectProp)).toBe('title');
     });
+    it('getDateFormat', () => {
+        expect(LocaleUtils.getDateFormat()).toBe("YYYY/MM/DD");
+        expect(LocaleUtils.getDateFormat("en-US")).toBe("MM/DD/YYYY");
+        expect(LocaleUtils.getDateFormat("it-IT")).toBe("DD/MM/YYYY");
+        expect(LocaleUtils.getDateFormat("nl-NL")).toBe("DD/MM/YYYY");
+        expect(LocaleUtils.getDateFormat("zh-ZH")).toBe("YYYY/MM/DD");
+        expect(LocaleUtils.getDateFormat("hr-HR")).toBe("DD/MM/YYYY");
+        expect(LocaleUtils.getDateFormat("pt-PT")).toBe("DD/MM/YYYY");
+    });
+    it('DATE_FORMATS', () => {
```

Comment:
This test is not useful at all.

Reply:
what if you accidentally erase the it-IT date format customization. it will use the default one.
this checks the presence of the customization for the some locales. i'll update the it description